src/Strategist/Strategist.cpp

```diff
@@ -9,13 +9,20 @@ void Strategist::onStart() {
     // Get initial gamestate information
     supply_total = BWAPI::Broodwar->self()->supplyTotal();
     chooseOpeningBuildOrder();
+    playDecision = PlayDecision::scout;
+    foundBase = false;
 }
 
 void Strategist::onFrame() {
     // Check if we need to add something to the queue
     if (!build_order_queue.empty()) {
         updateUnitQueue();
     }
+
+    if (!foundBase && foundEnemyBase()) {
```

Comment:
I'm not sure how beneficial `foundBase` is in this scenario, since you could also just do a comparison on `playerDecision == PlayerDecision::scout` instead.